app/file_sender.py

```diff
@@ -158,11 +159,24 @@ def start_file_sender(readiness_queue):
 
 def copy_files_to_sftp(file_paths: Collection[Path], remote_directory):
     with sftp.SftpUtility(remote_directory) as sftp_client:
-        logger.info('Copying files to SFTP remote', sftp_directory=sftp_client.sftp_directory)
+        logger.info('Copying files to SFTP remote and then GCP Bucket',
+                    sftp_directory=sftp_client.sftp_directory, gcp_bucket=Config.SENT_PRINT_FILE_BUCKET)
         for file_path in file_paths:
             sftp_client.put_file(local_path=str(file_path), filename=file_path.name)
-        logger.info(f'All {len(file_paths)} files successfully written to SFTP remote',
-                    sftp_directory=sftp_client.sftp_directory)
+            write_file_to_bucket(file_path)
```

Comment:
Not sure if this should be called within `copy_files_to_sftp`, feels like it should be called separately.